### PR TITLE
🌐 Add Indonesian translation for `docs/id/docs/tutorial/first-steps.md`

### PR DESCRIPTION
**Pull request by bas-baskara**

### CODE REVIEW

File: docs/id/docs/tutorial/first-steps.md [hunk 1]
```diff
@@ -0,0 +1,334 @@
+# First Steps
```
Comment (negadive) [inline docs/id/docs/tutorial/first-steps.md:1]: Great work @bas-baskara, shouldn't the index and title also need to be translated? [Here an example](https://github.com/tiangolo/fastapi/pull/1259/files#diff-36f5eba55d0de6739fbf682714cf8346d50ce364387653b6abdff54efce927ddR28) how they translate the index

**Review by negadive — CHANGES_REQUESTED** [review]

File: docs/id/docs/tutorial/first-steps.md (hunk 1 above)
Comment (bas-baskara) [reply docs/id/docs/tutorial/first-steps.md:1]: Hi @negadive

Based on the guidelines from https://fastapi.tiangolo.com/contributing/#existing-language: 

"Add a single pull request per page translated. That will make it much easier for others to review it."

So I translated this page for one PR. I've made another Pull Request to translate the index page.

**Comment by tiangolo** [comment]
📝 Docs preview for commit 1a0da27f098c57eaf907336f763a6d33eddbe71d at: https://649a113fc011270e85a55b86--fastapi.netlify.app

**Review by lieryan — APPROVED** [review]

**Review by lieryan — APPROVED** [review]
Generally looks good, just a couple minor improvements to fix some minor typos and some awkwardly phrased sentences.

**Comment by tiangolo** [comment]
As this PR had requested changes to be applied but has been inactive for a while, it's now going to be closed. But if there's anyone interested, feel free to create a new PR.

**Pull request closed without merging**